server.js

```diff
@@ -13,10 +13,10 @@ new WebpackDevServer(webpack(config), {
   stats: {
     colors: true,
   },
-}).listen(4242, 'localhost', function (err) {
+}).listen(process.env.PERFHTML_PORT || 4242, 'localhost', function (err) {
   if (err) {
     console.log(err);
   }
 
-  console.log('Listening at localhost:4242');
+  console.log(`Listening at localhost:${process.env.PERFHTML_PORT}`);
```

Comment:
You need `|| 4242` as well here, right ? or better, create a constant like `const port = process.env.PERFHTML_PORT || 4242` and use it everywhere.